nntrainer/compiler/ini_interpreter.cpp

```diff
@@ -118,26 +128,120 @@ std::shared_ptr<Layer> section2layer<PlainLayer>(dictionary *ini,
 }
 
 template <>
-std::shared_ptr<Layer> section2layer<BackboneLayer>(dictionary *ini,
-                                                    const std::string &sec_name,
-                                                    const AppContext &ac) {
-  return nullptr;
-}
+std::shared_ptr<Layer>
+section2layer<BackboneLayer>(dictionary *ini, const std::string &sec_name,
+                             const AppContext &ac,
+                             const std::string &backbone_file) {
+  std::string type;
+
+#if defined(ENABLE_NNSTREAMER_BACKBONE)
+  type = NNStreamerLayer::type;
+#endif
+
+/** TfLite has higher priority */
+#if defined(ENABLE_TFLITE_BACKBONE)
+  if (endswith(backbone_file, ".tflite")) {
+    type = TfLiteLayer::type;
+  }
+#endif
+
+  NNTR_THROW_IF(type.empty(), std::invalid_argument)
+    << FUNC_TAG
+    << "This nntrainer does not support external section: " << sec_name
+    << " backbone: " << backbone_file;
+
+  auto properties = section2properties(ini, sec_name);
+  properties.push_back("modelfile=" + backbone_file);
+  std::shared_ptr<ml::train::Layer> layer_ =
+    ac.createObject<ml::train::Layer>(type, properties);
 
-static std::shared_ptr<GraphRepresentation>
-section2graph(dictionary *ini, const std::string &sec_name) {
-  /// NYI!
-  return nullptr;
+  auto layer = std::static_pointer_cast<Layer>(layer_);
+  int status = layer->setName(sec_name);
+  throw_status(status);
+
+  return layer;
 }
 
-bool graphSupported(const std::string &backbone_name) {
-  /// NYI!
-  return true;
+/**
+ * @brief check if graph is supported
+ *
+ * @param backbone_name name of the backbone
+ * @return true if the file extension is supported to make a graph
+ * @return false if the file extension is not supported
+ */
+static bool graphSupported(const std::string &backbone_name) {
+  return endswith(backbone_name, ".ini");
 }
 
+/**
+ * @brief Get the Mergeable Graph object
+ *
+ * @param graph currently, extendGraph accepts
+ * std::vector<std::shared_ptr<Layer>>, so return in this format
+ * @param ini ini to parse property
+ * @param sec_name section name
+ * @return std::vector<std::shared_ptr<Layer>> mergeable graph
+ */
 std::vector<std::shared_ptr<Layer>>
-getMergeableGraph(std::shared_ptr<const GraphRepresentation>) {
-  return {};
+getMergeableGraph(std::shared_ptr<const GraphRepresentation> graph,
```

Comment:
This is a tweaked version of `model_loader::loadBackboneConfigIni`, the original function is being removed in upcoming PR